src/Activity/Log.php

```diff
@@ -225,11 +225,19 @@ public function save()
 			$this->set('details', json_encode($data));
 		}
 
+		$isNew = $this->isNew();
 		$result = parent::save();
 
 		if ($result)
 		{
-			Event::trigger('activity.onLogSave', [$this]);
+			if ($isNew)
+			{
+				Event::trigger('activity.onLogCreate', [$this]);
+			}
+			else
+			{
+				Event::trigger('activity.onLogSave', [$this]);
+			}
```

Comment:
Would it make sense to just pass the `$isNew` as a second argument to the `onLogSave` event and then listeners can choose how to respond?